### ISSUE TITLE
GrowMask node is needed

### ISSUE THREAD

**Issue by coreyryanhanson** [comfyanonymous/ComfyUI#1241]
The core nodes of the project don't have a way to expand masks.

I would argue that as the custom node ecosystem grows, implementations for this functionality would likely end up repeated over and over again, and that it belongs as a basic node in the mask group.

For example in my use case, I am developing an node that can auto generate masks based on perceived error from unscannable QR codes to allow fixing through image2image. It would be strange for my repo to include a basic node for growing masks to allow users more control over their selection.

I've just tested out an implementation and made a pull request.
https://github.com/comfyanonymous/ComfyUI/pull/1240

It uses scipy (already in project dependencies) and iterates through higher mask expansion values, but through my testing it is pretty quick for what it does.

I also tested an alternative implementation using only what is available in torch working directly in the tensors without conversion:

```
import torch.nn.functional as F

class GrowMaskConv2d:
    @classmethod
    def INPUT_TYPES(cls):
        return {
            "required": {
                "mask": ("MASK",),
                "expand": ("INT", {"default": 0, "min": 0, "max": MAX_RESOLUTION, "step": 1}),
            },
        }
    
    CATEGORY = "mask"

    RETURN_TYPES = ("MASK",)

    FUNCTION = "expand_mask"

    def expand_mask(self, mask, expand):
        if not expand:
            return (mask, )
        output = mask.clone().unsqueeze(0).unsqueeze(0)
        kernel_size = 2 * expand + 1
        padding = (kernel_size - 1) // 2
        dilation_kernel = torch.ones(1, 1, kernel_size, kernel_size).to(mask.device)
        dilation_kernel = dilation_kernel.float()
        output = F.conv2d(output, dilation_kernel, padding=padding)
        output = (output > 0).squeeze(0).squeeze(0)
        return (output,)
```
but that way was less efficient by roughly a magnitude of 10.
